# Goal
geometry_msgs/PoseStamped goal_pose
string object_id
string solver_name

---
# Result
bool success
string message

---
# Feedback
string current_phase
